packages/plugins/plugin-space/src/capabilities/intent-resolver.ts

```diff
@@ -324,6 +324,7 @@ export default ({ context, observability, createInvitationUrl }: IntentResolverO
                 props: {
                   target,
                   typename,
+                  onCreateObject,
```

Comment:
ok interesting -- this was what i was missing.
would prefer all `on` callbacks to be defined at the end.